### DISCUSSION TITLE
Support for Next 12

### DISCUSSION THREAD

**Discussion by asjadanis** [caching-tools/next-shared-cache#201] [Q&A] [answered]
Does the cache-handler library work with Next Version 12?

**Comment by better-salmon** [comment, marked as answer]
Hey, @asjadanis! The library currently only supports Next.js version 13.5.1 and newer. I did some research and found that in version 12.3.4, custom caching is possible, but the API of the `IncrementalCache` class is different. While I would love to make it work, it's not a priority milestone for me at the moment. However, if you're interested, try forking the library and adapting it to your needs.
